src/main/java/com/github/invictum/panels/FloatingPanel.java

```diff
@@ -0,0 +1,36 @@
+package com.github.invictum.panels;
+
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+public class FloatingPanel extends AbstractPanel {
+
```

Comment:
Looks like it is possible to skip constructor definition.
But initWith method must be redefined.